monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/BasePlugins.tsx

```diff
@@ -34,6 +34,13 @@ const getPluginsGridHeaders = (getRowActions) => [
   }
 ]
 
+enum pluginTypeHumanReadable {
```

Comment:
Do we need an enum for that?

Is there a chance that the type coming from the server will have a different name without the underscore?
E.g.: 
from server --> Credentials_Collector 
in ui --> Credentials Collector

can't we just replace the underscore with space and cap each word?

Reply:
Since these are values that aren't expected to change, I'd rather skip the extra logic. Even if something about how we display the name ever changes, this is easier to rework than trying to modify and debug the logic.

Reply:
Aren't expected by who? The table is a display element, nothing should expect it's columns to correspond to the plugin data.